### ISSUE TITLE
Initial accessibility support

### ISSUE THREAD

**Issue by mrego** [servo/servo#44399]
## Roadmap Issue

**Description:**
Accessibility is one of the main missing pieces in Servo, we want to work on adding accessibility support to Servo. This is a challenging project as we want to design a performant accessibility design. Main goal is to provide initial accessibility features to start exposing information to assistive technologies.

**Area:** Architecture

**People:** @Igalia, @alice, @delan

---
